app/authorization_test.go

```diff
@@ -4,6 +4,8 @@
 package app
 
 import (
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
```

Comment:
It getting a failure on compile because you are importing `require` but not using it.

```suggestion
```